.github/workflows/connectors_weekly_build.yml

```diff
@@ -41,4 +41,4 @@ jobs:
           gcp_gsm_credentials: ${{ secrets.GCP_GSM_CREDENTIALS }}
           git_branch: ${{ steps.extract_branch.outputs.branch }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          subcommand: "--show-dagger-logs connectors ${{ inputs.test-connectors-options || '--concurrency=3 --support-level=community' }} test"
+          subcommand: "--show-dagger-logs connectors ${{ inputs.test-connectors-options || '--concurrency=3 --metadata-query=\"(data.ab_internal.ql > 100) & (data.ab_internal.sl < 200)\"' }} test"
```

Comment:
This will reduce our weekly connector tests from over 200 down to ~70.

The question is should we test the 100 level ql community connectors weekly as well?

Reply:
if possible, it would still be nice to define this query as a variable that describes what it is for

Reply:
That's cool!  The parsing knows to convert the input into numbers and do math?!
Edit: Thanks, simple_eval!